### PR TITLE
fix: update dep

### PR DESCRIPTION
**Pull request by luan-dev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,67 +1,63 @@
...
+## [1.3.0](https://github.com/Asymmetrik/graphql-fhir/compare/1.3.0...1.2.4) (2021-09-15)
 
+### Update Dependencies
 
-## [1.2.3](https://github.com/Asymmetrik/graphql-fhir/compare/v1.2.2...v1.2.3) (2019-05-21)
+- upgrade to graphql 15.5.3
+  ** formatError to customFormatErrorFn
+  ** coerceValue to coerceInputValue
 
+TODO REMEMBER to add the following package back in before merge
```
Comment (ekoon121) [inline CHANGELOG.md:15]: you can remove this

File: src/config.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const VERSION = {
 	'1_0_2': '1_0_2',
 	'3_0_1': '3_0_1',
 	'4_0_0': '4_0_0',
+	//'4_0_1': '4_0_1',
```
Comment (ekoon121) [inline src/config.js:11]: Are you going to add your generated 4.0.1?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,67 +1,58 @@
...
-
+## [1.3.1](https://github.com/Asymmetrik/graphql-fhir/compare/1.2.4...1.3.1) (2021-09-17)
 
 ### Bug Fixes
 
-* updated package containing vulnerability ([0d99acd](https://github.com/Asymmetrik/graphql-fhir/commit/0d99acd))
-* upgrade packages ([#30](https://github.com/Asymmetrik/graphql-fhir/issues/30)) ([5cd0619](https://github.com/Asymmetrik/graphql-fhir/commit/5cd0619))
+- update deps ([35a2e9a](https://github.com/Asymmetrik/graphql-fhir/commit/35a2e9ad3a9d91e96238e20f40dc26428864a635))
 
+## [1.3.0](https://github.com/Asymmetrik/graphql-fhir/compare/1.3.0...1.2.4) (2021-09-15)
```
Comment (ekdeveloper) [inline CHANGELOG.md]: # clean up
Either use 1.2.4 to 1.3.1 and delete 1.2.4 to 1.3.0, or use 1.2.4 to 1.3.0 and 1.3.0 to 1.3.1

File: package.json [hunk 4]
```diff
@@ -1,71 +1,77 @@
...
+		"lint": "eslint \"src/**/*.js\"",
+		"changelog": "conventional-changelog -p angular -s -i CHANGELOG.md"
+	},
+	"prettier": {
+		"trailingComma": "all",
+		"singleQuote": true
+	},
+	"dependencies": {
+		"@asymmetrik/sof-graphql-invariant": "^2.0.0",
+		"@asymmetrik/sof-strategy": "^1.0.2",
```
Comment (ekdeveloper) [inline package.json]: https://github.com/Asymmetrik/node-fhir-server-core/pull/305/files
I think the other PR says sof-strategy as 1.1.5

File: package.json [hunk 5]
```diff
@@ -1,71 +1,77 @@
...
+	},
+	"devDependencies": {
+		"conventional-changelog-cli": "^2.1.1",
+		"coveralls": "^3.0.2",
+		"eslint": "^7.32.0",
+		"jest": "^27.2.0",
+		"prettier": "^2.4.1"
+	},
+	"resolutions": {
+		"graphql": "^15.3.0"
```
Comment (ekdeveloper) [inline package.json]: 15.6.0?

**Review by ekdeveloper — APPROVED** [review]
looks good

**Pull request merged**